modelcontextprotocol/server.py

```diff
@@ -320,9 +320,10 @@ def update_assets_tool(
         assets (Union[Dict[str, Any], List[Dict[str, Any]]]): Asset(s) to update.
             Can be a single UpdatableAsset or a list of UpdatableAsset objects.
         attribute_name (str): Name of the attribute to update.
-            Only "user_description" and "certificate_status" are supported.
+            Only "user_description", "certificate_status", and "readme" are supported.
         attribute_values (List[str]): List of values to set for the attribute.
             For certificateStatus, only "VERIFIED", "DRAFT", or "DEPRECATED" are allowed.
+            For readme, the value must be a valid HTML string without <html> and <body> tags OR a valid Markdown string.
```

Comment:
Check this as well

Reply:
Resolved via : https://github.com/atlanhq/agent-toolkit/pull/63/commits/4c78596e7cd2f739e532e1e25b482603b9bbf4a9